cashctrl_ledger/ledger_utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+This module provides utilities for handling and manipulating pandas DataFrames.
+"""
+
+import pandas as pd
+
+def df_to_str(target: pd.DataFrame) -> str:
```

Comment:
I am not convinced by the name `df_to_str`. It is too generic (`to_csv` is also converts a data frame to string) and does not convey the main point of the function, which is to translate the data frame to a unique, easily comparable form. Let's ask ChatGPT for better names.

Reply:
Here is the suggested names I like the most:

`df_to_consistent_str`
`standardize_df_to_str`
`df_to_standardized_str`

Reply:
I`ve predicted this vote)